apps/wpcom-block-editor/package.json

```diff
@@ -16,19 +16,25 @@
 	},
 	"homepage": "https://github.com/Automattic/wp-calypso",
 	"scripts": {
-		"clean": "npx rimraf dist",
-		"prebuild": "npm run clean",
-		"bundle": "webpack --config='../../packages/calypso-build/webpack.config.js' --env.WP='true' iframe-bridge-server='./src/iframe-bridge-server.js' tinymce='./src/tinymce.js'",
-		"build": "npm run bundle -- --output-path=./dist"
+		"bundle": "webpack --env.WP='true'",
+		"build:dev": "npm run bundle --",
+		"build:prod": "NODE_ENV=production npm run bundle --",
+		"build": "npm-run-all --parallel \"build:* -- {@}\" --"
 	},
 	"dependencies": {
 		"@wordpress/data": "4.4.0",
 		"@wordpress/blocks": "6.0.7",
+		"@wordpress/compose": "3.2.0",
+		"@wordpress/editor": "9.2.2",
 		"@wordpress/hooks": "2.0.5",
+		"@wordpress/rich-text": "3.2.2",
 		"@wordpress/url": "2.3.3",
 		"jquery": "^1.12",
 		"lodash": "4.17.11",
 		"react": "16.8.6",
 		"tinymce": "4.8.5"
+	},
+	"devDependencies": {
```

Comment:
A peculiarity of monorepo: `devDependencies` in non-root packages don't really mean anything. It's preferable to never include them, any devDependencies must be in the root package.

Reply:
Wouldn't that cause a conflict with the `import/no-extraneous-dependencies` ESLint rule?

<img width="1123" alt="Screen Shot 2019-04-18 at 11 07 14" src="https://user-images.githubusercontent.com/1233880/56332154-25625480-61ca-11e9-9fa3-3048dac742db.png">

Reply:
Might be. :-) It's not ideal — all I know it's causing trouble. See e.g.: https://github.com/Automattic/wp-calypso/pull/32383

cc @blowery